litmus-portal/frontend/src/components/Modal/FinishModal/index.tsx

```diff
@@ -10,18 +11,13 @@ function Icon() {
   return <img src="icons/finish.png" className={classes.mark} alt="mark" />;
```

Comment:
Could it be possible to use svgs instead!

Reply:
This is changed in Oum's PR #1864